### ISSUE TITLE
heap-resize reports incorrect amount value for type "release free pages"

### ISSUE THREAD

**Issue by Param-S** [eclipse/omr#1113]
Random cases where the value printed for heap-resize is not valid wrt "release free pages" type

<heap-resize id="6109" type="release free pages" space="tenure" **amount="18446744073416224768"** count="1" timems="4.125" reason="idle" timestamp="2017-05-08T03:15:29.279" />

from debugging, it seems to me that the condition where the FreeEntry size falls below the page size, is not taken into consideration by the following statement.

```
uintptr_t
MM_MemoryPoolAddressOrderedListBase::releaseFreeEntryMemoryPages(...)
{
...
uintptr_t totalFreePagesCount = (currentFreeEntry->getSize() - (addressBase - (uintptr_t)currentFreeEntry)) / pageSize;
...
}
```

**Comment by ashu-mehra** [comment]
I think this issue is fixed by https://github.com/eclipse/omr/pull/1114 and can be closed.

**Comment by charliegracie** [comment]
Closing as this was resolved in #1114
